### PR TITLE
feat(api): Show CIS docker benchmark results

### PR DESCRIPTION
**Pull request by FrimIdan**

### CODE REVIEW

File: api/swagger.yaml [hunk 1]
```diff
@@ -189,6 +189,48 @@ definitions:
       - CICD
       - RUNTIME
 
+  CISDockerBenchmarkLevelCount:
```
Comment (akpsgit) [inline api/swagger.yaml:192]: maybe CISDockerBenchmarkIssuesPerLevelCount
and to add description:
Number of CIS Docker benchmark issues for a specific level

File: api/swagger.yaml [hunk 2]
```diff
@@ -189,6 +189,48 @@ definitions:
...
+        format: integer
+      assessments:
+        type: 'array'
+        items:
+          $ref: '#/definitions/CISDockerBenchmarkAssessment'
+
+  CISDockerBenchmarkAssessment:
+    type: 'object'
+    properties:
+      code:
```
Comment (akpsgit) [inline api/swagger.yaml:224]: why code and level is duplicated from CISDockerBenchmarkCodeInfo?

File: api/swagger.yaml [hunk 3]
```diff
@@ -189,6 +189,48 @@ definitions:
...
+        type: 'string'
+      level:
+        type: 'number'
+        format: integer
+      filename:
+        type: 'string'
+      desc:
```
Comment (akpsgit) [inline api/swagger.yaml:231]: description

File: api/swagger.yaml (hunk 2 above)
Comment (FrimIdan) [reply api/swagger.yaml:226]: not sure, I just copied it from the runtime scan api https://github.com/cisco-open/kubei/blob/main/runtime_scan/api/swagger.yaml#L86

File: api/swagger.yaml (hunk 3 above)
Comment (FrimIdan) [reply api/swagger.yaml:233]: to be aligned with `CISDockerBenchmarkAssessment` in runtime_scan API

**Pull request closed without merging**